web/src/App.tsx

```diff
@@ -32,7 +32,7 @@ function App() {
     return (
       <CenterContainer>
         <Typography>Login in progress.</Typography>
-        <Progress.Dots color="primary" />
+        <Progress.Dots color='primary' />
```

Comment:
Vote for having double fnutts here, so changing `"jsxQuoteStyle": "single",` to `"jsxQuoteStyle": "double",` in `biome.json`.